tests/debug_bound_with_generic_type.rs

```diff
@@ -0,0 +1,23 @@
+use mockiato::{mockable, partial_eq};
+use std::fmt::Debug;
+
+#[mockable]
+trait Greeter<T, U>: Debug {
+    fn greet(&self, name: T) -> U;
+}
+
+#[derive(Eq, PartialEq)]
+struct Name;
+
+#[derive(Clone)]
+struct Greeting;
+
+#[test]
+fn debug_bound_with_generic_type_works() {
```

Comment:
jnferner lint failed: Test name contains the suffix `_works`
```
 fn debug_bound_with_generic_type_works() {
                                                          ^^^^^^^^
```
Help: Consider giving this test a more descriptive name.

Reply:
Addressed in e8c705a